symfony.lock

```diff
@@ -69,7 +69,7 @@
         ]
     },
     "php": {
-        "version": "7.3"
+        "version": "7.4"
```

Comment:
We are using 7.3.19

Reply:
Nice catch.
I'm using 7.4, and this is getting updated everytime... 😅

Reply:
Not sure if there is a way to avoid this… maybe setting the target platform in `composer.json` has an influence, I don't know.

Reply:
Why not using 7.4? Maybe we can adjust this in a separate PR first? Don’t forget to change .php-version file and .symfony.cloud.yaml

Reply:
I'll prefer if this is in another since I'm going on vacation and this PR should fix https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1083

And this is needed to finish https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1073